README.md

```diff
@@ -19,6 +19,11 @@ Deno shims for Node.js
 Commands:
 
 ```sh
+# get submodules it you did not clone them initially
+git submodule init
+git submodule update
+# upgrade or downgrade your deno binary to match deno source version in ./packages/shim-deno/third_party/deno
+deno upgrade --version 1.28.1
```

Comment:
Going to remove these two lines because it will likely get out of date and someone will be able to figure this out while working on the project because certain commands will/should error saying to use a specific Deno version.

```suggestion
```